.pytest.ini

```diff
@@ -1,10 +1,6 @@
-[aliases]
-test = pytest
-
-[tool:pytest]
+[pytest]
```

Comment:
I would put this in `pyproject.toml` to have fewer config files to manage/modify.